tests/api_fastapi/execution_api/routes/test_task_instances.py

```diff
@@ -716,6 +716,81 @@ def test_ti_runtime_checks_failure(self, client, session, create_task_instance):
 
         session.expire_all()
 
+    def test_ti_update_state_to_fail_with_asset_profile_extra(self, client, session, create_task_instance):
+        clear_db_assets()
+        clear_db_runs()
```

Comment:
This is done in setup teardown, not needed I think.

Reply:
These lines are in [`test_ti_update_state_to_success_with_asset_events`](https://github.com/apache/airflow/blob/6c536f52af5caf6170a4982b1ca44747f90863d1/tests/api_fastapi/execution_api/routes/test_task_instances.py#L311), but with removing them failed test, so I didn't remove them.